### ISSUE TITLE
bigip_command documentation is obsolete and confuses

### ISSUE THREAD

**Issue by slavkv** [F5Networks/f5-ansible#61]
##### ISSUE TYPE
 - Documentation Report

##### COMPONENT NAME
bigip_command

##### SUMMARY
Module documentation states you have to use the following format to connect:
http://f5-ansible.readthedocs.io/en/latest/modules/bigip_command_module.html
https://github.com/F5Networks/f5-ansible/blob/master/docs/modules/bigip_command_module.rst

- name: Load the default system configuration
  bigip_command:
      server: "bigip.localhost.localdomain"
      user: "admin"
      password: "admin"
      command: "tmsh load sys config default"
      validate_certs: "no"
  delegate_to: localhost

While this codes won't work and return errors as bigip_command is not using AnsibleModule at all.

Instead, you have to use a structure like that:
  - name: Run command
    bigip_command:
      provider: "{{ cli }}"
      commands: "tmsh list auth user"

Which is in mentioned only in https://github.com/F5Networks/f5-ansible/blob/master/roles/bigip_command/tasks/issue-54.yaml

In summary, it confuses and takes a lot of time to figure out the problem.

I would much appreciate if module documentation is updated on wiki and git which is already in bigip_command.py :)

Thanks for great work!

**Comment by caphrim007** [comment]
@slavkv thanks for the info. I will fix this

**Comment by caphrim007** [comment]
@slavkv I believe this is now fixed. docs should be rebuilding shortly
